src/System.Data.Common/ref/System.Data.Common.csproj

```diff
@@ -16,7 +16,7 @@
   <ItemGroup Condition="'$(TargetGroup)' == 'netfx'">
     <Reference Include="mscorlib" />
     <Reference Include="System" />
-    <Reference Include="System.Transactions" />
+    <Reference Include="System.Transactions.Local" />
```

Comment:
This still needs to remain System.Transactions as it is part of the netfx build configuration. That is what is breaking the current CI leg.